deployments/lisk.diamond.json

```diff
@@ -0,0 +1,76 @@
+{
+    "LiFiDiamond": {
+        "Facets": {
+            "0x57FdfF2e36De6c8a8Cde297B150Ae291132Eae8d": {
+                "Name": "DiamondCutFacet",
+                "Version": "1.0.0"
+            },
+            "0xA1a4d577709dC4A70CA38F1D41562fab3aD09D3f": {
+                "Name": "DiamondLoupeFacet",
+                "Version": "1.0.0"
```

Comment:
I actually wonder how you can have the versions in the diamond logs if the deploy log does not contain that information....that is utterly weird...

Reply:
It's generated by the deploy tool